crates/mun_codegen/src/snapshots/test__arithmetic_op_f32.snap

```diff
@@ -6,9 +6,9 @@ expression: "pub fn add(a: f32, b: f32) -> f32 { a + b }\npub fn subtract(a: f32
 ; ModuleID = 'main.mun'
 source_filename = "main.mun"
 
-%struct.MunTypeInfo = type { [16 x i8], i8 addrspace(4)*, i32, i8, i8 }
+%struct.MunTypeInfo = type { [16 x i8], i8*, i32, i8, i8 }
 
-@global_type_table = external global [1 x %struct.MunTypeInfo addrspace(4)*]
+@global_type_table = external global [1 x %struct.MunTypeInfo*]
```

Comment:
I noticed that
```
@global_type_table = external global [1 x %struct.MunTypeInfo*]
```
and
```
@global_type_table = constant [1 x %struct.MunTypeInfo*] [%struct.MunTypeInfo* @"type_info::<core::i64>"]
```
now differ between `global` and `constant`. Why is that?

P.S. This applies to all snapshots

Reply:
Thanks for the explanation.